README.md

```diff
@@ -131,3 +131,14 @@ The German government has asked SAP and Deutsche Telekom to develop the Corona-W
 [cwa-verification-portal]: https://github.com/corona-warn-app/cwa-verification-portal
 [cwa-verification-iam]: https://github.com/corona-warn-app/cwa-verification-iam
 [cwa-testresult-server]: https://github.com/corona-warn-app/cwa-testresult-server
+
+## Licensing
+
+Copyright (c) 2020 SAP SE or an SAP affiliate company.
+
+Licensed under the **Apache License, Version 2.0** (the "License"); you may not use this file except in compliance with the License.
+
+You may obtain a copy of the License from [here](./LICENSES/Apache-2.0.txt).
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the [LICENSE](./LICENSES/Apache-2.0.txt) for the specific language governing permissions and limitations under the License. 
+Please see the [detailed licensing information](https://api.reuse.software/info/github.com/corona-warn-app/cwa-app-ios) via the [REUSE Tool](https://reuse.software/) for more details.
```

Comment:
@Ein-Tim:
For better readability, please add an empty line between line 143 and 144..